src/EncompassRest/Loans/ATRQMBorrower.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace EncompassRest.Loans
 {
-    public sealed partial class ATRQMBorrower : ExtensibleObject
+    public sealed partial class ATRQMBorrower : ExtensibleObject, IIdentifiable
```

Comment:
if `ExtensibleObject` implements `IIdentifiable`, why is it part of each class as well?

Reply:
That is so that it will automatically make any `Id` properties the implicit interface implementation. And then we can always use explicit interface implementation for when we don't want to use the `Id` property such as for `CustomField` where we want to use `FieldName` instead. See [here](https://github.com/EncompassRest/EncompassRest/pull/85/files#diff-e54fcc398bcdcf57465d5817d8b90e28R11).